lib/htmlValidator.js

```diff
@@ -160,7 +172,11 @@ module.exports = function(app, callback) {
     next();
   });
 
-  Promise.all([promise]).then(() => {
+  promise.then(function (reason) {
```

Comment:
Btw @Autre31415 `reason` seems to be the standard according to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch) not sure if that matters as far as readability/styling is concerned?